client/routes/search.js

```diff
@@ -49,14 +53,15 @@ module.exports = (app) => {
     const documents = data['content']['documents'];
     log(
       'fetch',
-      `Received documents from ranker. Execution time ${(Date.now() - startTime) /
+      `Received documents from ranker. Execution time ${(Date.now() - fetchStartTime) /
         1000} seconds.`
     );
 
     if (req.user != undefined) {
       User.findById(req.user._id, (err, user) => {
         if (err) log('error', 'Unable to find user');
         else {
+          clickLogger.setCurrQueryTime(Date.now() - queryStartTime);
```

Comment:
This is going to cause the same issue as with setting the current query inside of clickLogger. Probably for the best just to write this info in a different file b/c this won't work with multiple clients.

Reply:
I was mistaken here